tools/reviewer/web/src/app/dashboard/reviews-panel/diff/code-block/code-block.component.ts

```diff
@@ -0,0 +1,178 @@
+import { ChangeDetectorRef, Component, Input, OnInit } from '@angular/core';
+
+import { Comment, Thread } from '@/shared';
+import { HighlightService } from '@/shared/services';
+import { DiffService } from '../diff.service';
+
+export interface Line {
+  code: string; // Original code of the line
+  highlightedCode: string; // Code with html tags
+  isOpen: boolean;  // Does the line have a placeholder for comments?
```

Comment:
how about a variable name like `hasPlaceholder`?

Reply:
I like it